omod/src/main/java/org/openmrs/module/registrationapp/RegistrationAppUiUtils.java

```diff
@@ -105,4 +106,35 @@ public static void validateLatitudeAndLongitudeIfNecessary(PersonAddress address
 			}
 		}
 	}
+	
+	/**
+	 * Get the defined custom registration appId from the global property value registrationcore.customRegistrationAppId defined by
+	 *
+	 * @see RegistrationCoreConstants#GP_CUSTOM_REGISTRATION_APPID
+	 *
+	 * @return the custom appId or null if none is defined
+	 */
+	public static String getCustomRegistrationAppId() {
+		// check if there is a custom registration app registered then use that
+		String customRegistratonAppId = Context.getAdministrationService().getGlobalProperty(RegistrationCoreConstants.GP_CUSTOM_REGISTRATION_APPID);
+		if (org.apache.commons.lang3.StringUtils.isNotBlank(customRegistratonAppId)) {
+			return customRegistratonAppId;
+		}
+		
+		return null;
+	}
+	
```

Comment:
How about merging these into one method that returns the configured appId, if none is set you return the default. If you want to keep both methods for some reason, I would rename the second one `getDefaultRegistrationAppId()`